x/amm/spec/06_swap_txs_reordering.md

```diff
@@ -0,0 +1,35 @@
+# Swap transactions reordering
+
+To provide lowest slippage as possible on the block, module sort the swap requests into a specific order.
+
+This removes possibility of MEV attack on swap transactions.
```

Comment:
@jelysn While the spec mentions that this setup removes the possibility of an MEV attack, it doesn't explain how it achieves that. It would be helpful to outline how the reordering prevents MEV attacks explicitly.

Reply:
Okay